### PR TITLE
fix/trigger-resubmit-test

### PR DESCRIPTION
**Pull request by brian-mulier-p**

### CODE REVIEW

File: core/src/main/java/io/kestra/core/runners/Worker.java [hunk 1]
```diff
@@ -73,6 +73,7 @@ public class Worker implements Runnable, AutoCloseable {
     @Getter
     private final Map<Long, AtomicInteger> metricRunningCount = new ConcurrentHashMap<>();
 
+    @Getter
```
Comment (loicmathieu) [inline core/src/main/java/io/kestra/core/runners/Worker.java:76]: Can you add `@VisibleForTesting` to know that it's only used for testing purpose?

File: jdbc/src/test/java/io/kestra/jdbc/runner/JdbcHeartbeatTest.java [hunk 2]
```diff
@@ -175,10 +178,17 @@ void triggerResubmit() throws Exception {
         AtomicReference<WorkerTriggerResult> workerTriggerResult = new AtomicReference<>(null);
         workerTriggerResultQueue.receive(either -> {
             workerTriggerResult.set(either.getLeft());
+            countDownLatch.countDown();
```
Comment (loicmathieu) [inline jdbc/src/test/java/io/kestra/jdbc/runner/JdbcHeartbeatTest.java:181]: This count down latch seems to be unused now

**Review by loicmathieu — CHANGES_REQUESTED** [review]

File: jdbc/src/test/java/io/kestra/jdbc/runner/JdbcHeartbeatTest.java (hunk 2 above)
Comment (brian-mulier-p) [reply jdbc/src/test/java/io/kestra/jdbc/runner/JdbcHeartbeatTest.java:181]: It's used L197 to await till a triggerResult is received to ensure that the resubmit worked properly and that the second worker took the task

File: core/src/main/java/io/kestra/core/runners/Worker.java (hunk 1 above)
Comment (brian-mulier-p) [reply core/src/main/java/io/kestra/core/runners/Worker.java:77]: Done

**Review by loicmathieu — APPROVED** [review]
LGTM

**Review by loicmathieu — APPROVED** [review]
LGTM

**Pull request merged**